src/r/utils/ensureArray.js

```diff
@@ -2,9 +2,9 @@
  * @param {*} value
  * @returns {*[]}
  */
-const ensureArray = (value) => {
+const ensureArray = value => {
```

Comment:
Doesn't the AirBnB JS Styleguide describe that there should always be parentheses unless the arrow function returns a value directly (eg `a => a` without parentheses, everything else with parentheses)?

Reply:
I think `prettier` might still be overriding this one. It's annoying 😁